app/controllers/daily_menus_controller.rb

```diff
@@ -1,5 +1,13 @@
 class DailyMenusController < ApplicationController
   def index
-    @date = DateTime.now.strftime("%d %b %Y")
+    @daily_menus = DailyMenu.last(10)
+  end
+
+  def show
+    @daily_menu = DailyMenu.find(params[:id])
+    @user_order = FoodItem.joins(:orders, :daily_menu).where(order_items: {user_id: current_user.id}, 
+                                                             daily_menu: {id: params[:id]})
```

Comment:
you can use just `daily_menu_id` instead of ` daily_menu: {id: params[:id]}` and get rid of joining `daily_menu`
another way to write it is just
`@user_order = @daily_menu.food_items.joins(:daily_menu).where(order_items: { user: current_user })`

Reply:
and it looks like `@user_orders` is better name

Reply:
> and it looks like `@user_orders` is better name

but actually, this variable holds only one order with many order items of the current user